batch/batch/inst_coll_config.py

```diff
@@ -118,6 +120,8 @@ def __init__(self, name, boot_disk_size_gb, max_instances, max_live_instances):
     def convert_requests_to_resources(self, machine_type, storage_bytes):
         # minimum storage for a GCE instance is 10Gi
         storage_gib = max(10, requested_storage_bytes_to_actual_storage_gib(storage_bytes))
```

Comment:
I think you need to catch the `None` after the call to `requested_storage_bytes_to_actual_storage_gib` before passing it to max. When I run `python3 -c 'max(10, None)'`

```python3
Traceback (most recent call last):
  File "<string>", line 1, in <module>
TypeError: '>' not supported between instances of 'NoneType' and 'int'
```

I also personally get a linter error on this line on main as it is now using pyright, saying

```
■ No overloads for "max" match the provided arguments    Argument types: (Literal[10], Unknown | int | None)
```

Maybe this would be a good thing for me to add to our lint checks?

Reply:
good catch.